parl/remote/scripts.py

```diff
@@ -249,6 +251,8 @@ def start_master(port, cpu_num, monitor_port, debug, log_server_port_range):
 @click.option(
     "--cpu_num", type=int, help="Set number of cpu manually. If not set, it will use all "
     "cpus of this machine.")
+@click.option(
```

Comment:
gpu_ids -> gpu.

help='comma separated list of GPU(s) to use. Default to use all available GPUs'